sonic-xcvrd/xcvrd/xcvrd.py

```diff
@@ -1477,13 +1523,7 @@ def task_worker(self):
                               else:
                                  self.log_notice("{} Successfully configured Tx power = {}".format(lport, tx_power))
 
-                        appl = self.get_cmis_application_desired(api, host_lanes, host_speed)
-                        if appl < 1:
-                            self.log_error("{}: no suitable app for the port".format(lport))
-                            self.port_dict[lport]['cmis_state'] = self.CMIS_STATE_FAILED
-                            continue
-
-                        need_update = self.is_cmis_application_update_required(api, host_lanes, host_speed)
+                        need_update = self.is_cmis_application_update_required(api, host_lane_count, host_lanes_mask, host_speed)
```

Comment:
definition of this function takes 'channel' argument name, can we keep both same?

Reply:
I have fixed this now.